src/account-info/AccountDetails.js

```diff
@@ -82,12 +90,16 @@ class AccountDetails extends PureComponent<Props> {
             <RawLabel style={componentStyles.statusText} text={localTime} />
           </View>
         )}
+        {botOwner !== undefined && (
+          <ZulipButton text={botOwner.full_name} onPress={this.handlePress} secondary />
```

Comment:
I can't think of a way of showing the name is related to bot_owner and at the same time indicating that it's touchable, any suggestions @ray-kraesig @agrawal-d 
Or the button can show a static text 'View bot owner' , which on clicking takes to the profile of bot owner.

Reply:
I like the way the webapp does it.
![image](https://user-images.githubusercontent.com/7714968/80564428-6ca13900-8a0b-11ea-97f1-a8e8dfdb18b9.png)
This can also be done in the app.